### PR TITLE
Fix #275, Fix #305, Fix #306, update ID linking scheme

### LINKED ISSUE #275: ID Linking Scheme Figure is out of date
Figure 3 "ID Linking Scheme" (§[3.2.2](https://aomediacodec.github.io/iamf/#data))  should be updated based on the change in #233.

The Codec Config OBU no longer has audio_element_ids. Now the Audio Elements have codec_config_id which is how they are matched with a Codec Config OBU.

The figure still has the old fields and the arrows are in the wrong direction between the Codec Config OBU and Audio Element OBU.

Also small editorial inconsistency. One arrow from the Codec Config points to the green Audio Element box. But the other arrow points to the white audio_element_id under the Audio element box. Make sure to have the arrows consistent after updating the main issue above (these arrows will be affected by the change above),

### LINKED ISSUE #305: `mix_loudness_info()` is not defined in the Spec
It appears only once in §[3.2.2](https://aomediacodec.github.io/iamf/#data).

Probably should be `loudness_info()`.

### LINKED ISSUE #306: In Figure 3 Audio Elements should hold codec config IDs.
In Figure 3 in §[3.2.2](https://aomediacodec.github.io/iamf/#data), it currently shows the Codec Configs holding audio element IDs. It should be the other way around.

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/320.html" title="Last updated on Mar 15, 2023, 5:32 AM UTC (1378832)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/320/4621c48...1378832.html" title="Last updated on Mar 15, 2023, 5:32 AM UTC (1378832)">Diff</a>

### CODE REVIEW

**Review by jwcullen — APPROVED** [review]

**Review by yeroro — CHANGES_REQUESTED** [review]

**Comment by tdaede** [comment]
Mergeable once yeroro's comments are addressed.

File: index.bs [hunk 1]
```diff
@@ -407,28 +407,32 @@ The IA sequence supports the description of multiple audio substreams and algori
 - <dfn noexport>Temporal Delimiter OBU</dfn> identifies the temporal units.
 
 
-The below figure shows the linking scheme among [=obu_id=]s in obu_header and ids in obu payload.
+The below figure shows the linking scheme among IDs in obu_header or ids in obu payload.
```
[the remark this answers is not in the document]
Comment (sunghee-hwang) [reply index.bs]: The header of Audio frame obu can have one of audio substream ids between ID0 to ID21.

File: index.bs [hunk 2]
```diff
@@ -407,28 +407,32 @@ The IA sequence supports the description of multiple audio substreams and algori
...
+- Mix Presentation OBU with mix_presentation_id  = 21 is saying:
+	- There are two Audio Elements(audio_element_id = 11 and 12) which need to be mixed. The audio_element_id = 11 and the audio_element_id = 12 are linked to the Audio Element OBUs with audio_element_id = 11 and audio_element_id = 12, respectively.
+		- There is Paramter Block OBU with parameter_id = 32 to be used for mixing of the Audio Element with audio_element_id = 11. 
+		- There is Paramter Block OBU with parameter_id = 33 to be used for mixing of the Audio Element with audio_element_id = 12.
+	- There is Parameter Block OBU with parameter_id = 34 to be used for mixing of the two Audio Elements.
+- Audio Element OBU with audio_element_id = 11 is saying:
+	- This audio element has been coded by using Codec Config OBU with codec_config_id = 0. 
+	- There are two Substreams (audio_substream_id = 0 and 1) which are composing of this audio element. The audio_substream_id = 0 and the audio_substream_id = 1 are linked to the Audio Frame OBUs with audio_substream_id = 0 and audio_substream_id = 1(i.e. obu_type = OBU_IA_Audio_Frame_ID0 and obu_type = OBU_IA_Audio_Frame_ID1), respectively.
```
[the remark this answers is not in the document]
Comment (yeroro) [reply index.bs:424]: Please address.

[the remark this answers is not in the document]
Comment (sunghee-hwang) [reply index.bs:424]: > Please address.

Please clarify what it needs to be addressed.

[the remark this answers is not in the document]
Comment (yeroro) [reply index.bs:424]: As mentioned in the previous comment https://github.com/AOMediaCodec/iamf/pull/320#discussion_r1133759961

Please replace "which are composing of" with "in".

**Review by yeroro — APPROVED** [review]

**Pull request merged**